tests/test_src/test_wrappers/test_wrappers.cpp

```diff
@@ -149,3 +150,50 @@ TEST(CryptoWrapper, getSignature)
     EXPECT_EQ(util::stringToHex(signature), expectedSignature);
 
 }
+
+TEST(ClientWrapper, get_validClient)
+{
+    wrapper::http::Client client("https://api.elrond.com");
+    wrapper::http::Result res =  client.get("/address/erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz");
+
+    EXPECT_FALSE(res.error);
+    EXPECT_EQ(res.status, STATUS_CODE_OK);
+    EXPECT_FALSE(res.body.empty());
+    EXPECT_TRUE(res.body.find("success"));
+    EXPECT_EQ(res.statusMessage, STATUS_MSG_OK);
+}
+
+TEST(ClientWrapper, get_invalidClient)
+{
+    wrapper::http::Client client("https://api.elronddd.com");
+    wrapper::http::Result res =  client.get("/address/erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz");
+
+    EXPECT_TRUE(res.error);
+    EXPECT_NE(res.status, STATUS_CODE_OK);
+    EXPECT_TRUE(res.body.empty());
+    EXPECT_NE(res.statusMessage, STATUS_MSG_OK);
+}
+
+TEST(ClientWrapper, post_validClient)
+{
+    wrapper::http::Client client("https://api.elrond.com");
+    wrapper::http::Result res =  client.post("/transaction/send", "test");
+
+    EXPECT_FALSE(res.error);
+    EXPECT_EQ(res.status, 400); // 400 = BAD REQUEST
+    EXPECT_FALSE(res.body.empty());
+    EXPECT_TRUE(res.body.find("error"));
+    EXPECT_NE(res.statusMessage, STATUS_MSG_OK);
+}
+
+TEST(ClientWrapper, post_invalidClient)
+{
+    wrapper::http::Client client("https://api.elronddd.com");
```

Comment:
If you really need this kind of tests, please use a GOOD domain & a BAD subdomain instead (since the domain `elronddd` is not owned by us, and thus the test is prone to external actions).

Reply:
Yes, that's a good point, I've refactored the tests to check for valid domain and bad subdomains instead